cli/script/command-executor.ts

```diff
@@ -676,6 +826,29 @@ function getPackageString(packageObject: Package): string {
         (packageObject.description ? wordwrap(70)("\n" + chalk.green("Description: ") + packageObject.description): "");
 }
 
+function getPackageMetricsString(packageObject: PackageWithMetrics): string {
+    if (!packageObject || !packageObject.metrics) {
+        return "";
+    }
+    
+    var percentString: string = packageObject.metrics.activePercent === 100.0 ? "100" : packageObject.metrics.activePercent.toPrecision(2) + "%";
+    var numPending: number = packageObject.metrics.downloaded - packageObject.metrics.installed - packageObject.metrics.failed;
+    var returnString: string = chalk.green("Active: ") + percentString + " (" + packageObject.metrics.active + ")";
+    if (packageObject.metrics.installed || numPending) {
+        returnString += "\n" + chalk.green("Installs: ") + ("" + packageObject.metrics.installed);
+    }
+    ;
+    if (numPending) {
+        returnString += " (" + numPending + " pending)";
+    }    
+    
+    if (packageObject.metrics.failed) {
+        returnString += "\n" + chalk.green("Failures: ") + chalk.red(packageObject.metrics.failed + "");
```

Comment:
What do you think about calling this "Rollbacks" instead? I think it more accurately reflects what it represents since we can't say for sure that it was actually a failure. It could have been a dev error (by not calling notifyApplicarionReady), a timing issue (the device died at the perfect moment), etc.